<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeaderCommand</class>
 <widget class="QFrame" name="HeaderCommand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>25</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="buttonLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="leftSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>28</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="addBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="../wposgui.qrc">
       <normaloff>:/controls/icons/list-add.png</normaloff>:/controls/icons/list-add.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="../wposgui.qrc">
       <normaloff>:/controls/icons/list-remove.png</normaloff>:/controls/icons/list-remove.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="searchEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string extracomment="Enter Barcode to Scan"/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="rightSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../wposgui.qrc"/>
 </resources>
 <connections/>
</ui>
